examples/gallery/lines/hlines_vlines.py

```diff
@@ -0,0 +1,112 @@
+"""
+Horizontal and vertical lines
+=============================
+
+The :meth:`pygmt.Figure.hlines` and :meth:`pygmt.Figure.vlines`
+methods allow to plot horizontal and vertical lines in
+Cartesian, Geographic and Polar projections.
+"""
+
+# %%
+# In Cartesian projections lines are plotted as straight lines.
+
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.basemap(
+    region=[0, 10, 0, 10], projection="X10c/10c", frame=["+thlines Cartesian", "af"]
+)
+
+# add a red horizontal line at y=9 without specifying x limits
+fig.hlines(y=9, pen="1.5p,red3", label="Line 1")
+# add a gray dashed horizontal line at y=8 with x limits at 2 and 8
+fig.hlines(y=8, xmin=2, xmax=8, pen="1.5p,gray30,-", label="Line 2")
+# add two salmon-colored horizontal lines at y=6 and y=7 both with x limits at 3 and 7
+fig.hlines(y=[6, 7], xmin=3, xmax=7, pen="1.5p,salmon", label="Line 3")
+# add two black dotted horizontal lines at y=4 and y=5 both with x limits at 4 and 9
+fig.hlines(y=[4, 5], xmin=4, xmax=9, pen="1.5p,black,.", label="Line 4")
+# add two blue horizontal lines at y=2 and y=3 with different x limits
+fig.hlines(y=[2, 3], xmin=[0, 1], xmax=[7, 7.5], pen="1.5p,dodgerblue3", label="Line 5")
+fig.legend(position="JBR+jBR+o0.2c", box="+gwhite+p1p")
+
+fig.shift_origin(xshift="w+2c")
+
+fig.basemap(
+    region=[0, 10, 0, 10], projection="X10c/10c", frame=["+tvlines Cartesian", "af"]
+)
+# add a red vertical line at x=1 without specifying y limits
+fig.vlines(x=1, pen="1.5p,red3", label="Line 1")
+# add a gray dashed vertical line at x=2 with y limits at 2 and 8
+fig.vlines(x=2, ymin=2, ymax=8, pen="1.5p,gray30,-", label="Line 2")
+# add two salmon-colored vertical lines at x=3 and x=4 both with y limits at 3 and 7
+fig.vlines(x=[3, 4], ymin=3, ymax=7, pen="1.5p,salmon", label="Line 3")
+# add two black dotted vertical lines at x=5 and x=6 both with y limits at 4 and 9
+fig.vlines(x=[5, 6], ymin=4, ymax=9, pen="1.5p,black,.", label="Line 4")
+# add two blue vertical lines at x=7 and x=8 with different y limits
+fig.vlines(x=[7, 8], ymin=[0, 1], ymax=[7, 7.5], pen="1.5p,dodgerblue3", label="Line 5")
+fig.legend()
+
+fig.show()
+
+# %%
+# The same can be done for Geographic projections where horizontal means
+# the lines are plotted as parallels along constant latitude and vertical
+# lines are plotted as parallels along constant longitude.
+
+fig = pygmt.Figure()
+
+fig.basemap(region="g", projection="R15c", frame=["+thlines Geographic", "af"])
+# add a red horizontal line at a latitude of 70°N without specifying longitude limits
+fig.hlines(y=70, xmin=0, xmax=360, pen="1.5p,red3", label="line1")
```

Comment:
Would make the legend entries consistent, i.e. either upper or lower case starting and either with or without a white space between "line" and the related number.

Reply:
Done